Library/Homebrew/cmd/readall.rb

```diff
@@ -31,7 +31,7 @@ def readall
     taps = if ARGV.named.empty?
       Tap
     else
-      [Tap.fetch(ARGV.named.first)]
+      ARGV.named.map { |t| Tap.fetch(t) }
```

Comment:
The help at the top needs updated to match the fact it can take multiple taps. Otherwise 👍